### PR TITLE
AcerIncorporated.AcerProductRegistration version 2.0.3030.0

### PR DESCRIPTION
**Pull request by Charlie-TW**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/87072)

### CODE REVIEW

File: manifests/a/AcerIncorporated/AcerProductRegistration/2.0.3030.0/AcerIncorporated.AcerProductRegistration.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AcerIncorporated.AcerProductRegistration
+PackageVersion: 2.0.3030.0
+Installers:
+- InstallerUrl: http://10.36.22.111:8180/HungCharlie/prodreginstallers/-/raw/main/2b39f60c02594a838bf4de7851532b59.msix?inline=false
```
Comment (Trenly) [inline manifests/a/AcerIncorporated/AcerProductRegistration/2.0.3030.0/AcerIncorporated.AcerProductRegistration.installer.yaml:7]: Please use a public URL and not a URL which is internal to your domain

File: manifests/a/AcerIncorporated/AcerProductRegistration/2.0.3030.0/AcerIncorporated.AcerProductRegistration.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: AcerIncorporated.AcerProductRegistration
+PackageVersion: 2.0.3030.0
+PackageLocale: en-US
+Publisher: Acer Incorporated
+PackageName: Acer Product Registration
+License: Copyright (c) Acer Incorporated
```
Comment (Trenly) [inline manifests/a/AcerIncorporated/AcerProductRegistration/2.0.3030.0/AcerIncorporated.AcerProductRegistration.locale.en-US.yaml:9]: A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Uses private URL which isn't accessible to the public;

**Pull request closed without merging**